adaptive_memory_v4.0.py

```
@@ -0,0 +1,2126 @@
+                cache = {}
+                if await asyncio.to_thread(os.path.exists, cache_file):
+                    try:
+                        def _load():
+                            with open(cache_file, 'r') as f:
+                                return json.load(f)
+                        cache = await asyncio.to_thread(_load)
+                    except Exception as e:
+                        logger.warning(f"Error loading embedding cache for batch store: {e}")
+                
+                # Update cache with new embeddings
+                for memory_id, text, embedding in zip(ids, texts, embeddings):
```

Ruff (0.14.10)

Loop control variable `text` not used within loop body

Rename unused `text` to `_text`

(B007)